distribution/docker/docker-compose.yml

```diff
@@ -46,7 +46,7 @@ services:
       - ZOO_MY_ID=1
 
   coordinator:
-    image: apache/incubator-druid
```

Comment:
I'm not familiar with how our Docker builds get built. Does changing this require ASF Infra to do anything? If so, please add a sub-task for https://issues.apache.org/jira/browse/INFRA-19609.

Reply:
I created https://issues.apache.org/jira/browse/INFRA-19648